src/training/day3/Tasks.java

```diff
@@ -0,0 +1,117 @@
+package training.day3;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Tasks {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        // zadacha 1
+        System.out.println("Task 1");
+        int[] numbers = new int[]{10, 2, 1, -1, 5, 9, 7, 6, 5, -5};
+        task1(numbers);
+
+        // zadacha 2
+        System.out.println("Task 2");
+        String word1 = "Plovdiv";
+        String word2 = "lov";
+        if (word1.contains(word2)) {
+            String result = word1.replace(word2, "");
+            System.out.println(result);
+        }
+
+        // zadacha 3
+        System.out.println("Task 3");
+        System.out.println(reversing("I love Java"));
+
+        // zadacha 4
+        System.out.println("Task 4");
+        System.out.println(initials("Kristiyan Zheliazkov"));
+
+        // zadacha 5
+        System.out.println("Task 5");
+        ArrayList<String> arrlist = new ArrayList();
+        System.out.println("Enter words:");
+        arrlist.add(scanner.nextLine());
+        arrlist.add(scanner.nextLine());
+        arrlist.add(scanner.nextLine());
+        arrlist.add(scanner.nextLine());
+        swappingElements(arrlist);
+
+        for (int i = 0; i < arrlist.size(); ++i) {
+            System.out.println((String) arrlist.get(i));
+        }
+
+    }
+
+    static void task1(int[] numbers) {
```

Comment:
Logic in the method can be extracted in separate methods. And these methods must have suitable names.